src/app/descriptor-languages/Service.ts

```diff
@@ -1,5 +1,9 @@
 import { ExtendedDescriptorLanguageBean } from 'app/entry/extendedDescriptorLanguage';
 import { SourceFile, ToolDescriptor, Workflow } from 'app/shared/swagger';
+import { Dockstore } from '../shared/dockstore.model';
+
+const SERVICE_DOCUMENTATION_URL =
+  Dockstore.DOCUMENTATION_URL + '/getting-started/getting-started-with-services.html?highlight=service#services';
```

Comment:
I don't think you want the `?highlight=service#services` -- that gets appended when you navigate from the search page.

Reply:
removed the excess url ending